libs/design-system/code-block/src/lib/code-block.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="code-block">
```

Comment:
Please remove this wrapper div and put the code-block styles directly inside  `:host { /* Styles goes here */ }` instead